docs/porting/nuget-packages.md

```diff
@@ -1,5 +1,36 @@
-# 🔧 Porting NuGet packages
+# Porting NuGet Packages
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+This article covers how to get your ported .NET Framework code to NuGet.  Although the packaging format has not changed, your code has not fundamentally changed with .NET Core code (compared with .NET Framework code), there are different avenues with which you can package your code.
+
+## Prerequisities
+
+This article assumes that you have already ported your code to .NET Core.  If you have not done that, you can follow [Porting Your Libraries](libraries.md), as it contains relevant information if your code is a library or a framework.
+
+If you wish to use cross-platform tools, you'll have to download the [.NET CLI](http://aka.ms/dotnetcoregs)
```

Comment:
I'd avoid future tense here and add the missing period in the end. Maybe "you must download the [.NET CLI](http://aka.ms/dotnetcoregs) first." 
In addition to that, when the user goes to this link, he/she will get to the Getting started page. Nowhere there we mention .NET CLI. Are they going to be confused when they don't find .NET CLI mentioned there? Do we need to explain this a bit more here or make it explicit on the Getting started page that tools==.NET CLI?